### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by NobodyKJ**
Make rest Api and longtoshort working 
Screen Shot 2021-12-07 at 12 10 24 AM
Screen Shot 2021-12-16 at 10 48 20 PM
Screen Shot 2021-12-16 at 10 49 32 PM

### CODE REVIEW

**Review by LeonQC — DISMISSED** [review]
Good job

File: src/main/java/com/briefly/backenddesign/controller/UrlController.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        urlRepository.save(updateUrl);
+
+        return ResponseEntity.ok(updateUrl);
+    }
+
+
+
+    // build delete employee REST API
+    @DeleteMapping("{id}")
+    public ResponseEntity<HttpStatus> deleteEmployee(@PathVariable long id){
```
Comment (LeonQC) [inline src/main/java/com/briefly/backenddesign/controller/UrlController.java:50]: No need delete

File: src/main/java/com/briefly/backenddesign/controller/UrlController.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    // build get employee by id REST API
+    @GetMapping("{id}")
+    public ResponseEntity<Url> getUrlById(@PathVariable  long id){
+        Url url = urlRepository.findById(id).orElseThrow();
+        return ResponseEntity.ok(url);
+    }
+
+    // build update employee REST API
+    @PutMapping("{id}")
+    public ResponseEntity<Url> updateUrl(@PathVariable long id,@RequestBody Url urlDetails) {
```
Comment (LeonQC) [inline src/main/java/com/briefly/backenddesign/controller/UrlController.java]: No need update

**Review by LeonQC — DISMISSED** [review]

File: src/main/java/com/briefly/backenddesign/controller/UrlController.java (hunk 2 above)
Comment (NobodyKJ) [reply src/main/java/com/briefly/backenddesign/controller/UrlController.java]: Aready remove this

File: src/main/java/com/briefly/backenddesign/controller/UrlController.java (hunk 1 above)
Comment (NobodyKJ) [reply src/main/java/com/briefly/backenddesign/controller/UrlController.java:50]: I keep this function to remove back-end data at easier, will remove at end of this class

**Review by LeonQC — APPROVED** [review]

**Pull request merged**